### ISSUE TITLE
dialog isn't accessible via keyboard

### ISSUE THREAD

**Issue by blindfish3** [ArthurClemens/dialogic#47]
I could be wrong; but on the demo page I was unable to use the keyboard to move focus into the dialog or trigger the buttons in the opened dialogs.  Keyboard support is a requirement to meet accessibility standards.  From what I gather it's also a tricky problem to solve in notification dialogs; which is why I was hoping to find an existing solution :sweat_smile:

**Comment by ArthurClemens** [comment]
I don't think there is anything specific to dialogic that would prevent this. Dialogic manages dialog/notification state, but what is displayed - and how you provide interacting with the content - is up to you.

**Comment by blindfish3** [comment]
Then this is presumably the result of the implementation in the example?  I would recommend updating the example so that it is keyboard accessible.  From a quick test of the demo some people may discard the component without digging into the internals.  Otherwise you could mention in the docs that accessibility is implementation dependent.
